gvmtools/cli.py

```diff
@@ -124,36 +124,39 @@ def main():
     else:
         protocol_class = Gmp
 
-    with protocol_class(connection, transform=transform) as protocol:
-
-        if args.protocol == PROTOCOL_GMP:
-            # Ask for password if none are given
-            if args.gmp_username and not args.gmp_password:
-                args.gmp_password = getpass.getpass(
-                    'Enter password for ' + args.gmp_username + ': '
-                )
-
-            if args.gmp_username:
-                protocol.authenticate(args.gmp_username, args.gmp_password)
-
-        try:
-            if args.duration:
-                starttime = time.time()
-
-            result = protocol.send_command(xml)
-
-            if args.duration:
-                duration = time.time() - starttime
-                print('Elapsed time: {} seconds'.format(duration))
-            elif args.pretty:
-                pretty_print(result)
-            else:
-                print(result)
-
-        except Exception as e:  # pylint: disable=broad-except
-            print(e, file=sys.stderr)
-            sys.exit(1)
-
+    try:
+        with protocol_class(connection, transform=transform) as protocol:
+
+            if args.protocol == PROTOCOL_GMP:
+                # Ask for password if none are given
+                if args.gmp_username and not args.gmp_password:
+                    args.gmp_password = getpass.getpass(
+                        'Enter password for ' + args.gmp_username + ': '
+                    )
+
+                if args.gmp_username:
+                    protocol.authenticate(args.gmp_username, args.gmp_password)
+
+            try:
+                if args.duration:
+                    starttime = time.time()
+
+                result = protocol.send_command(xml)
+
+                if args.duration:
+                    duration = time.time() - starttime
+                    print('Elapsed time: {} seconds'.format(duration))
+                elif args.pretty:
+                    pretty_print(result)
+                else:
+                    print(result)
+
+            except Exception as e:  # pylint: disable=broad-except
+                print(e, file=sys.stderr)
+                sys.exit(1)
```

Comment:
Could you remove this now unnecessary try/catch block too? Please also add an entry in the CHANGELOG.md file.

Reply:
Without it, then the exception will thrown.  Or was something changed in the gvm python lib?

Reply:
You are catching the expection already at the outside of the width statement. I don't see a reason to keep the inner try/except block too. Or I am missing something?

Reply:
Ah, now I understand it. You mean the inner exception.
Yes, you are right, because the result are the same. The app can't  do anything more.
I will remove the inner one.

Reply:
Yes the inner try/except block :-) Thanks a lot!

Reply:
Removed, but I can't see why the codecov/patch test fails.

Reply:
Just ignore it. It calculates the code coverage of the touched code only.